converters/toZigbee.js

```diff
@@ -1261,6 +1267,13 @@ const converters = {
                 val = utils.getKey(constants.thermostatControlSequenceOfOperations, value, value, Number);
             }
             await entity.write('hvacThermostat', {ctrlSeqeOfOper: val});
+            // NOTE: read the value instead of returning state
+            //       this way the internal device hvacThermostat.ctrlSeqeOfOper
+            //       we track is updated.
+            //
+            //       this is needed because some device, e.g. SMaBiT AV2010/32 have dynamic
+            //       expose data based on this attribute
+            await entity.read('hvacThermostat', ['ctrlSeqeOfOper']);
```

Comment:
To prevent an unnecessary read, we can also update to value directly here, example: https://github.com/Koenkk/zigbee-herdsman-converters/blob/ee8db717cd831dc807a5b70ee57358d51b177e74/converters/fromZigbee.js#L7531 (`msg.endpoint` becomes `entity`)